src/librustdoc/passes/collect_trait_impls.rs

```diff
@@ -213,6 +213,6 @@ impl BadImplStripper {
     }
 
     fn keep_impl_with_def_id(&self, did: &FakeDefId) -> bool {
```

Comment:
Passing a reference seems weird here  - why not pass by value?

Reply:
it passed a reference before so I kept it, but passing by value probably makes more sense yeah